requirements.in

```diff
@@ -1,3 +1,7 @@
 -e git://github.com/mbertheau/python-emarsys@master#egg=python-emarsys
 # lowest supported django version
 Django<1.8
+pytest
+pytest-django
+mock
+jsonfield
```

Comment:
Would also be great if you could squash the requirements changes with their corresponding commits.